exe/mauth-client

```diff
@@ -4,7 +4,6 @@ $LOAD_PATH.unshift File.expand_path('../lib', File.dirname(__FILE__))
 
 require 'rubygems'
```

Comment:
not in the diff but we can do without this now, can't we?

Reply:
oh yeah, I think so. this code is hurting my eyes so I'm going to run rubocop also.